pkg/testutil/test/command.go

```diff
@@ -105,8 +107,18 @@ func (gc *GenericCommand) Run(expect *Expected) {
 			iCmdCmd.Stdin = tty
 			iCmdCmd.Stdout = tty
 			iCmdCmd.Stderr = tty
-			defer pty.Close()
-			defer tty.Close()
```

Comment:
please don't move `defer tty.Close()`

Reply:
Why?

Reply:
not mandatory but you can just defer closing the tty before the goroutines loop